container/default.json

```diff
@@ -74,10 +74,10 @@
+    "contents": "This page is currently a work in progress and will be updated gradually as more components are added to the web site.<br><h2 id='logging-in'>Logging in</h2>When attempting to access certain features of Flexible Interactive Automation (FIA), you will be presented with a login page.<br><img src='/res/images/login.png' style='max-width:1000px; height:auto;'><br>You can login by entering your username and password provided by the <a href='https://www.isis.stfc.ac.uk/Pages/User-Office.aspx'>ISIS User Office</a>.<br><br>Once you've successfully logged in, certain pages will now be available. To the top left of the screen you will see a menu icon of three horizontal lines. This opens and closes the sidebar menu. If it's not already open, click the icon and you will see two options to click on under the 'Reductions' tab.<br><img src='/res/images/sidebar-menu.png' style='max-width:1000px; height:auto;'><h2 id='instruments'> Instruments</h2>The <a href='#'>Instruments</a> tab gives you an overview of all of ISIS's instruments. These are presented as cards which can be expanded.<br><img src='/res/images/instruments-page.png' style='max-width:1000px; height:auto;'><br><strong>Expanding a card</strong><br>If you expand a card you will get more information about an instrument as well as two buttons which lead to the data viewer page and the page for live reductions.<br><img src='/res/images/expanded-instrument-card.png' style='max-width:1000px; height:auto;'><h2 id='reduction-history'> Reduction history</h2>The <a href='#'>Reduction history</a> tab allows you to browse a list of previous reductions specific to a chosen instrument.<br><img src='/res/images/reduction-history.png' style='max-width:1000px; height:auto;'><br>All of the columns can be sorted ascending or descending. In the top right of the page there is a dropdown widget to quickly switch between instruments.<h2 id='data-viewer'> Data viewer</h2>Clicking on a reduction takes you to the data viewer page for that specific reduction which shows a summary of that reduction as well as any scripts and graphs of the data.<br><img src='/res/images/data-viewer.png' style='max-width:1000px; height:auto;'>"
```

Comment:
```suggestion
    "contents": "This page is currently a work in progress and will be updated gradually as more components are added to the web site.<br><h2 id='logging-in'>Logging in</h2>When attempting to access certain features of Flexible Interactive Automation (FIA), you will be presented with a login page.<br><img src='/res/images/login.png' style='max-width:1000px; height:auto;'><br>You can login by entering your username and password provided by the <a href='https://www.isis.stfc.ac.uk/Pages/User-Office.aspx'>ISIS User Office</a>.<br><br>Once you've successfully logged in, certain pages will now be available. To the top left of the screen you will see a menu icon of three horizontal lines. This opens and closes the sidebar menu. If it's not already open, click the menu icon and you will see two options to click on under the 'Reductions' tab.<br><img src='/res/images/sidebar-menu.png' style='max-width:1000px; height:auto;'><h2 id='instruments'> Instruments</h2>The <a href='#'>Instruments</a> tab gives you an overview of all of ISIS's instruments. These are presented as cards which can be expanded.<br><img src='/res/images/instruments-page.png' style='max-width:1000px; height:auto;'><br><strong>Expanding a card</strong><br>If you expand a card you will get more information about an instrument as well as two buttons which lead to the data viewer page and the page for live reductions.<br><img src='/res/images/expanded-instrument-card.png' style='max-width:1000px; height:auto;'><h2 id='reduction-history'> Reduction history</h2>The <a href='#'>Reduction history</a> tab allows you to browse a list of previous reductions specific to a chosen instrument.<br><img src='/res/images/reduction-history.png' style='max-width:1000px; height:auto;'><br>All of the columns can be sorted ascending or descending. In the top right of the page there is a dropdown widget to quickly switch between instruments.<h2 id='data-viewer'> Data viewer</h2>Clicking on a reduction takes you to the data viewer page for that specific reduction which shows a summary of that reduction as well as any scripts and graphs of the data.<br><img src='/res/images/data-viewer.png' style='max-width:1000px; height:auto;'>"
```

Reply:
just changing "icon" to "menu icon" when discussing the sidebar menu icon for expanding the sidebar menu